client/packages/invoices/src/Returns/api/hooks/utils/useReturnsApi.ts

```diff
@@ -0,0 +1,17 @@
+import { useAuthContext, useGql } from '@openmsupply-client/common';
+import { getReturnsQueries } from '../../api';
+import { getSdk } from '../../operations.generated';
+
+export const useReturnsApi = () => {
+  const { storeId } = useAuthContext();
+  const keys = {
+    base: () => ['returns'] as const,
+    count: () => [...keys.base(), 'count'] as const,
+    detail: (id: string) => [...keys.base(), storeId, id] as const,
+    newReturns: () => [...keys.base(), storeId, 'newReturns'] as const,
+  };
+
+  const { client } = useGql();
+  const queries = getReturnsQueries(getSdk(client), storeId);
+  return { ...queries, storeId, keys };
+};
```

Comment:
This is good though. I think I'll use this branch as a base for my one so I can start with something that seems accessible.